docker/images/pinot/etc/jmx_prometheus_javaagent/configs/server.yml

```diff
@@ -31,14 +31,14 @@ rules:
     tableType: "$2"
     topic: "$3"
     partition: "$4"
-- pattern: "\"org.apache.pinot.common.metrics\"<type=\"ServerMetrics\", name=\"pinot.server.highestStreamOffsetConsumed.([^\\.]*?)_(OFFLINE|REALTIME)\\-(.+)\\-(\\w+)\"><>(\\w+)"
-  name: "pinot_server_highestStreamOffsetConsumed_$5"
-  cache: true
-  labels:
-    table: "$1"
-    tableType: "$2"
-    topic: "$3"
-    partition: "$4"
+#- pattern: "\"org.apache.pinot.common.metrics\"<type=\"ServerMetrics\", name=\"pinot.server.highestStreamOffsetConsumed.([^\\.]*?)_(OFFLINE|REALTIME)\\-(.+)\\-(\\w+)\"><>(\\w+)"
+#  name: "pinot_server_highestStreamOffsetConsumed_$5"
+#  cache: true
+#  labels:
+#    table: "$1"
+#    tableType: "$2"
+#    topic: "$3"
+#    partition: "$4"
```

Comment:
do we still need this? can we delete?

Reply:
+1

Reply:
and also can we remove other commented code?